docs/user/discover.asciidoc

```diff
@@ -82,23 +82,22 @@ that shows all the documents that match your search.
 By default, the document table includes a column for the time field and a column that lists all other fields in the document.
 You’ll modify the document table to display your fields of interest.
 
-. In the sidebar, enter `ma` to in the search field to find the `manufacturer` field.
+. In the sidebar, enter `ma` in the search field to find the `manufacturer` field.
 +
 [role="screenshot"]
 image:images/discover-sidebar-available-fields.png[Fields list that displays the top five search results, width=50%]
++
+NOTE: You can use wildcards in field searches.  For example, `goe*dest` finds `geo.dest` and `geo.src.dest`.
 
 . In the *Available fields* list, click `manufacturer` to view its most popular values.
 +
 **Discover** shows the top 10 values and the number of records used to calculate those values.
 
-. Click image:images/add-icon.png[Add icon] to toggle the field into the document table.
-+
-[role="screenshot"]
-image::images/discover-add-icon.png[How to add a field as a column in the table, width="50%"]
+. Drag the field into the document table, and **Discover** creates a column for the field.
```

Comment:
it might be worth to mention that still using the plus icon allows a quick adding of columns to the table